src/utils/formatCoffeeScriptAst.ts

```diff
@@ -70,7 +72,7 @@ function shouldTraverse(value) {
  * CoffeeScript AST nodes are always instances of a custom class, so use the
  * constructor name to distinguish between node children and non-node children.
  */
-function isNode(value) {
+function isNode(value: {}): boolean {
```

Comment:
I'm not sure there's really a better type here, although maybe `any` and silencing the lint rule would be better. This is reflectively walking CoffeeScript AST nodes to show in the repl, and doing some runtime heuristics to figure out what is and isn't a node so it knows when to traverse.